app/models/event.rb

```diff
@@ -22,7 +23,6 @@ class Event < ApplicationRecord
   scope :by_follows, ->(follows) { where( organizer_id: follows ) }
 
   def self.available_for(user)
-    # Event.joins(:inscriptions).where.not("user_id != ?", user.id).or(where("user_id == ?", "")).where.not(organizer_id: user.id).where(  "min_age <= ?", user.age )
     Event.where.not(organizer_id: user.id).and( Event.where( "min_age <= ?", user.age )) - user.inscriptions.map(&:event) - Event.all.select{|event| event.completed?}
```

Comment:
why this change is here? maybe a banana? Moreover, the removed line seems better than the one you are going to keep